collector/Cargo.toml

```diff
@@ -41,11 +41,3 @@ winapi = { version = "0.3", features = ["winerror"] }
 [[bin]]
```

Comment:
Now that `collector` is the only binary in this directory, can this `[[bin]]` section be removed?

Reply:
The main binary cannot be moved to the bin directory yet, since it contains mod declarations, that's why I still left it here. I will refactor it some more in another PR to move things to lib.rs, then I can move the binary and remove the bin section completely.

Reply:
AIUI if you only have one binary then it doesn't need to be in the `bin` directory and you don't need the `[[bin]]` section?

Reply:
Ah, you're right, I forgot that it's called `main.rs`, in that case it will indeed work.